src/Shared/E2ETesting/selenium-config.json

```diff
@@ -1,7 +1,7 @@
 {
   "drivers": {
     "chrome": {
-      "version" : "91.0.4472.77"
+      "version" : "91.0.4472.19"
```

Comment:
Usually we try to match this version and the SeleniumWebDriverChromeDriverVersion: https://www.nuget.org/packages/Selenium.WebDriver.ChromeDriver/91.0.4472.1900-beta at https://github.com/dotnet/aspnetcore/blob/main/eng/Versions.props#L250.

Reply:
It's non-obvious what the relationship is between those two things, and I'd like to unblock builds

Reply:
The relationship is mainly to the version of Chrome installed. Kestrel's Interop.FunctionalTests.csproj _should_ fail w/ the current version or will real soon now. Plz fix if possible though separate PR may be fine.

Reply:
Someone will need to add the package to the dotnet-public feed